packages/oauth/oauth-client/src/oauth-client.ts

```diff
@@ -425,6 +437,7 @@ export class OAuthClient extends CustomEventTarget<OAuthClientEventMap> {
 
       const server = await this.serverFactory.fromIssuer(
         stateData.iss,
+        stateData.authMethod ?? 'legacy',
```

Comment:
Using the literal `"legacy"` if the `authMethod` is not defined (because `stateData` was created through an old version of this lib)